src/components/Method/Method.js

```diff
@@ -23,7 +23,7 @@ export default class Method extends PureComponent {
           </h3>
           <div>
             {description && <Description description={description} />}
-            {parameters && this.renderParameters(parameters)}
+            {parameters && <Parameters parameters={parameters} classes={classes} /> }
```

Comment:
We don't need to pass `classes` down to this component (there are none for it and if there were, they should be styled by the component, not the partent).